borgmatic/commands/hook.py

```diff
@@ -0,0 +1,7 @@
+import subprocess
+
+
+def exec_cmd(config):
```

Comment:
Even though I don't have a formal coding style convention written down in this project, I generally try to err on the side of verbosity rather than terseness in variable and function naming. So perhaps expand this out to `execute_command()` or similar. Or maybe even better: `execute_hook()`, because that's more specific to the domain in question (hooks). And in fact there may be multiple commands executed by a single hook!